prediction_market_agent_tooling/markets/omen/omen_contracts.py

```
@@ -163,10 +163,10 @@ def mergePositions(
         self,
         api_keys: APIKeys,
         collateral_token_address: ChecksumAddress,
-        parent_collection_id: HexStr,
         conditionId: HexBytes,
         index_sets: t.List[int],
         amount: Wei,
+        parent_collection_id: HexStr = build_parent_collection_id(),
```

Ruff

Do not perform function call `build_parent_collection_id` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)